binaries/opae.io/opae/io/utils.py

```diff
@@ -132,14 +132,28 @@ def vfio_init(pci_addr, new_owner='', force=False):
     load_driver('vfio-pci')
 
     print('Binding {} to vfio-pci'.format(msg))
-    new_id = '/sys/bus/pci/drivers/vfio-pci/new_id'
-    try:
-        with open(new_id, 'w') as outf:
-            outf.write('{} {}'.format(vid_did[0], vid_did[1]))
-    except OSError as exc:
-        if exc.errno != errno.EEXIST:
-            LOG.error(f'Cannot write new_id to vfio-pci for: {msg}')
+
+    # On Linux kernel >= 3.15, use driver_override to bind the
+    # vfio-pci driver, which ensures the driver binds only to
+    # the given function. On older kernels, the driver will
+    # instead bind to all functions of the associated device.
```

Comment:
Is the behavior unique to older kernels, or is it just by nature of having used new_id rather than driver_override?